### ISSUE TITLE
Images are broken on homepage

### ISSUE THREAD

**Issue by hadley** [jrnold/ggthemes#137]
https://jrnold.github.io/ggthemes/

**Comment by espinielli** [comment]
What worked for me was to have the following `fig.path` setup in the my README.Rmd

````{r, echo = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>",
  fig.path = "man/figures/README-"
)
````
Probably copied from something I found on the web, like this https://github.com/ThinkR-open/remedy/blob/master/README.Rmd

**Comment by KayleeDavisGitHub** [comment]
Thanks @espinielli! I sent a pull request yesterday that should fix the README :)
Just waiting on a maintainer to approve that.

**Comment by gdickens** [comment]
Just flagging this issue again. The images are broken at: https://jrnold.github.io/ggthemes/

**Comment by jrnold** [comment]
Fixed by #162

**Comment by espinielli** [comment]
Still broken
